/home/jlb638/.conda/envs/clip/lib/python3.11/site-packages/transformers/utils/hub.py:124: FutureWarning: Using `TRANSFORMERS_CACHE` is deprecated and will be removed in v5 of Transformers. Use `HF_HOME` instead.
  warnings.warn(
2024-06-05 22:32:01.126325: I tensorflow/core/platform/cpu_feature_guard.cc:210] This TensorFlow binary is optimized to use available CPU instructions in performance-critical operations.
To enable the following instructions: AVX2 AVX512F FMA, in other operations, rebuild TensorFlow with the appropriate compiler flags.
2024-06-05 22:32:11.473541: W tensorflow/compiler/tf2tensorrt/utils/py_utils.cc:38] TF-TRT Warning: Could not find TensorRT
Detected kernel version 3.10.0, which is below the recommended minimum of 5.5.0; this can cause the process to hang. It is recommended to upgrade the kernel to the minimum version or higher.
wandb: Currently logged in as: jlbaker361. Use `wandb login --relogin` to force relogin
wandb: wandb version 0.17.0 is available!  To upgrade, please run:
wandb:  $ pip install wandb --upgrade
wandb: Tracking run with wandb version 0.16.1
wandb: Run data is saved locally in /scratch/jlb638/wandb/wandb/run-20240605_223230-xxxjm0s3
wandb: Run `wandb offline` to turn off syncing.
wandb: Syncing run divine-pond-70
wandb: ‚≠êÔ∏è View project at https://wandb.ai/jlbaker361/ddpo-appendix
wandb: üöÄ View run at https://wandb.ai/jlbaker361/ddpo-appendix/runs/xxxjm0s3
Loading pipeline components...:   0%|          | 0/6 [00:00<?, ?it/s]Loading pipeline components...:  17%|‚ñà‚ñã        | 1/6 [00:04<00:24,  4.81s/it]Loading pipeline components...:  33%|‚ñà‚ñà‚ñà‚ñé      | 2/6 [00:09<00:19,  4.99s/it]Loading pipeline components...:  50%|‚ñà‚ñà‚ñà‚ñà‚ñà     | 3/6 [00:10<00:08,  2.77s/it]Loading pipeline components...:  67%|‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñã   | 4/6 [00:10<00:03,  1.93s/it]Loading pipeline components...: 100%|‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà| 6/6 [00:10<00:00,  1.80s/it]
Loading pipeline components...:   0%|          | 0/6 [00:00<?, ?it/s]Loading pipeline components...:  17%|‚ñà‚ñã        | 1/6 [00:00<00:00,  9.25it/s]Loading pipeline components...:  33%|‚ñà‚ñà‚ñà‚ñé      | 2/6 [00:00<00:01,  3.99it/s]Loading pipeline components...: 100%|‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà| 6/6 [00:00<00:00, 10.94it/s]
Loading pipeline components...:   0%|          | 0/6 [00:00<?, ?it/s]Loading pipeline components...:  17%|‚ñà‚ñã        | 1/6 [00:00<00:00,  9.54it/s]Loading pipeline components...:  33%|‚ñà‚ñà‚ñà‚ñé      | 2/6 [00:00<00:00,  4.14it/s]Loading pipeline components...: 100%|‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà| 6/6 [00:00<00:00, 11.26it/s]
Traceback (most recent call last):
  File "/cache/home/jlb638/Desktop/clipcreat/appendix.py", line 87, in <module>
    main(args)
  File "/cache/home/jlb638/Desktop/clipcreat/appendix.py", line 53, in main
    model_dict={
               ^
  File "/cache/home/jlb638/Desktop/clipcreat/appendix.py", line 54, in <dictcomp>
    model: get_pipeline(model,accelerator.device) for model in args.model_list
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/cache/home/jlb638/Desktop/clipcreat/appendix.py", line 36, in get_pipeline
    pipeline.sd_pipeline.unet.to(device)
  File "/home/jlb638/.conda/envs/clip/lib/python3.11/site-packages/torch/nn/modules/module.py", line 1160, in to
    return self._apply(convert)
           ^^^^^^^^^^^^^^^^^^^^
  File "/home/jlb638/.conda/envs/clip/lib/python3.11/site-packages/torch/nn/modules/module.py", line 810, in _apply
    module._apply(fn)
  File "/home/jlb638/.conda/envs/clip/lib/python3.11/site-packages/torch/nn/modules/module.py", line 810, in _apply
    module._apply(fn)
  File "/home/jlb638/.conda/envs/clip/lib/python3.11/site-packages/torch/nn/modules/module.py", line 810, in _apply
    module._apply(fn)
  [Previous line repeated 7 more times]
  File "/home/jlb638/.conda/envs/clip/lib/python3.11/site-packages/torch/nn/modules/module.py", line 833, in _apply
    param_applied = fn(param)
                    ^^^^^^^^^
  File "/home/jlb638/.conda/envs/clip/lib/python3.11/site-packages/torch/nn/modules/module.py", line 1158, in convert
    return t.to(device, dtype if t.is_floating_point() or t.is_complex() else None, non_blocking)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
torch.cuda.OutOfMemoryError: CUDA out of memory. Tried to allocate 20.00 MiB. GPU 0 has a total capacty of 11.91 GiB of which 9.62 MiB is free. Including non-PyTorch memory, this process has 11.90 GiB memory in use. Of the allocated memory 11.54 GiB is allocated by PyTorch, and 114.08 MiB is reserved by PyTorch but unallocated. If reserved but unallocated memory is large try setting max_split_size_mb to avoid fragmentation.  See documentation for Memory Management and PYTORCH_CUDA_ALLOC_CONF
wandb: - 0.004 MB of 0.004 MB uploadedwandb: \ 0.004 MB of 0.023 MB uploadedwandb: üöÄ View run divine-pond-70 at: https://wandb.ai/jlbaker361/ddpo-appendix/runs/xxxjm0s3
wandb: Synced 6 W&B file(s), 0 media file(s), 0 artifact file(s) and 0 other file(s)
wandb: Find logs at: /scratch/jlb638/wandb/wandb/run-20240605_223230-xxxjm0s3/logs
Exception in thread Exception in thread IntMsgThrNetStatThr:
:
Traceback (most recent call last):
Traceback (most recent call last):
  File "/home/jlb638/.conda/envs/clip/lib/python3.11/threading.py", line 1038, in _bootstrap_inner
  File "/home/jlb638/.conda/envs/clip/lib/python3.11/threading.py", line 1038, in _bootstrap_inner
        self.run()self.run()

  File "/home/jlb638/.conda/envs/clip/lib/python3.11/threading.py", line 975, in run
  File "/home/jlb638/.conda/envs/clip/lib/python3.11/threading.py", line 975, in run
        self._target(*self._args, **self._kwargs)self._target(*self._args, **self._kwargs)

  File "/home/jlb638/.conda/envs/clip/lib/python3.11/site-packages/wandb/sdk/wandb_run.py", line 300, in check_internal_messages
  File "/home/jlb638/.conda/envs/clip/lib/python3.11/site-packages/wandb/sdk/wandb_run.py", line 268, in check_network_status
        self._loop_check_status(self._loop_check_status(

  File "/home/jlb638/.conda/envs/clip/lib/python3.11/site-packages/wandb/sdk/wandb_run.py", line 224, in _loop_check_status
  File "/home/jlb638/.conda/envs/clip/lib/python3.11/site-packages/wandb/sdk/wandb_run.py", line 224, in _loop_check_status
        local_handle = request()local_handle = request()

                                      ^^^^^^^^^^^^^^^^^^

  File "/home/jlb638/.conda/envs/clip/lib/python3.11/site-packages/wandb/sdk/interface/interface.py", line 756, in deliver_network_status
  File "/home/jlb638/.conda/envs/clip/lib/python3.11/site-packages/wandb/sdk/interface/interface.py", line 764, in deliver_internal_messages
        return self._deliver_network_status(status)return self._deliver_internal_messages(internal_message)

                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
^  File "/home/jlb638/.conda/envs/clip/lib/python3.11/site-packages/wandb/sdk/interface/interface_shared.py", line 484, in _deliver_network_status
^^^^^^^^^    ^^return self._deliver_record(record)^

   File "/home/jlb638/.conda/envs/clip/lib/python3.11/site-packages/wandb/sdk/interface/interface_shared.py", line 490, in _deliver_internal_messages
         return self._deliver_record(record) 
       ^ ^ ^ ^ ^ ^ ^ ^ ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
^  File "/home/jlb638/.conda/envs/clip/lib/python3.11/site-packages/wandb/sdk/interface/interface_shared.py", line 437, in _deliver_record
^^^    ^^handle = mailbox._deliver_record(record, interface=self)^
^ 
  File "/home/jlb638/.conda/envs/clip/lib/python3.11/site-packages/wandb/sdk/interface/interface_shared.py", line 437, in _deliver_record
         handle = mailbox._deliver_record(record, interface=self) 
           ^ ^ ^ ^ ^ ^ ^ ^ ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
^  File "/home/jlb638/.conda/envs/clip/lib/python3.11/site-packages/wandb/sdk/lib/mailbox.py", line 455, in _deliver_record
^^^^^^^
  File "/home/jlb638/.conda/envs/clip/lib/python3.11/site-packages/wandb/sdk/lib/mailbox.py", line 455, in _deliver_record
        interface._publish(record)interface._publish(record)

  File "/home/jlb638/.conda/envs/clip/lib/python3.11/site-packages/wandb/sdk/interface/interface_sock.py", line 51, in _publish
  File "/home/jlb638/.conda/envs/clip/lib/python3.11/site-packages/wandb/sdk/interface/interface_sock.py", line 51, in _publish
        self._sock_client.send_record_publish(record)self._sock_client.send_record_publish(record)

  File "/home/jlb638/.conda/envs/clip/lib/python3.11/site-packages/wandb/sdk/lib/sock_client.py", line 221, in send_record_publish
  File "/home/jlb638/.conda/envs/clip/lib/python3.11/site-packages/wandb/sdk/lib/sock_client.py", line 221, in send_record_publish
        self.send_server_request(server_req)self.send_server_request(server_req)

  File "/home/jlb638/.conda/envs/clip/lib/python3.11/site-packages/wandb/sdk/lib/sock_client.py", line 155, in send_server_request
  File "/home/jlb638/.conda/envs/clip/lib/python3.11/site-packages/wandb/sdk/lib/sock_client.py", line 155, in send_server_request
        self._send_message(msg)self._send_message(msg)

  File "/home/jlb638/.conda/envs/clip/lib/python3.11/site-packages/wandb/sdk/lib/sock_client.py", line 152, in _send_message
  File "/home/jlb638/.conda/envs/clip/lib/python3.11/site-packages/wandb/sdk/lib/sock_client.py", line 152, in _send_message
        self._sendall_with_error_handle(header + data)self._sendall_with_error_handle(header + data)

  File "/home/jlb638/.conda/envs/clip/lib/python3.11/site-packages/wandb/sdk/lib/sock_client.py", line 130, in _sendall_with_error_handle
  File "/home/jlb638/.conda/envs/clip/lib/python3.11/site-packages/wandb/sdk/lib/sock_client.py", line 130, in _sendall_with_error_handle
        sent = self._sock.send(data)sent = self._sock.send(data)

                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

BrokenPipeErrorBrokenPipeError: : [Errno 32] Broken pipe[Errno 32] Broken pipe

srun: error: pascal010: task 0: Exited with exit code 1
